gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceConfigurationKeys.java

```diff
@@ -0,0 +1,118 @@
+  public static final boolean DEFAULT_COMPLIANCE_JOB_SIMULATE = false;
+
+  public static final String GOBBLIN_COMPLIANCE_JOB_CLASS = COMPLIANCE_PREFIX + ".job.class";
+  public static final String GOBBLIN_COMPLIANCE_DATASET_FINDER_CLASS = COMPLIANCE_PREFIX + ".dataset.finder.class";
+
+  public static final String GOBBLIN_COMPLIANCE_PROXY_USER = COMPLIANCE_PREFIX + ".proxy.user";
+  public static final boolean GOBBLIN_COMPLIANCE_DEFAULT_SHOULD_PROXY = false;
+  public static final String GOBBLIN_COMPLIANCE_SHOULD_PROXY = COMPLIANCE_PREFIX + ".should.proxy";
+  public static final String GOBBLIN_COMPLIANCE_SUPER_USER = COMPLIANCE_PREFIX + ".super.user";
+
+  public static final String RETENTION_VERSION_FINDER_CLASS_KEY = COMPLIANCE_PREFIX + ".retention.version.finder.class";
+
+  public static final String RETENTION_SELECTION_POLICY_CLASS_KEY =
+      COMPLIANCE_PREFIX + ".retention.selection.policy.class";
+
+  public static final String DATASET_SELECTION_POLICY_CLASS = COMPLIANCE_PREFIX + ".dataset.selection.policy.class";
+  public static final String DEFAULT_DATASET_SELECTION_POLICY_CLASS = "gobblin.compliance.HivePartitionDatasetPolicy";
+
+  public static final String PURGER_COMMIT_POLICY_CLASS = COMPLIANCE_PREFIX + ".purger.commit.policy.class";
+  public static final String DEFAULT_PURGER_COMMIT_POLICY_CLASS = "gobblin.compliance.purger.HivePurgerCommitPolicy";
+
+  public static final String RETENTION_VERSION_CLEANER_CLASS_KEY =
+      COMPLIANCE_PREFIX + ".retention.version.cleaner.class";
+
+  public static final String RESTORE_DATASET = COMPLIANCE_PREFIX + ".restore.dataset";
+
+  public static final String RESTORE_POLICY_CLASS = COMPLIANCE_PREFIX + ".restore.policy.class";
+
+  public static final String DATASET_TO_RESTORE = COMPLIANCE_PREFIX + ".dataset.to.restore";
+
+  public static final String PURGE_POLICY_CLASS = COMPLIANCE_PREFIX + ".purge.policy.class";
+
+  public static final String VALIDATION_POLICY_CLASS = COMPLIANCE_PREFIX + ".validation.policy.class";
+
+  // Name of the column in the compliance id table containing compliance id.
+  public static final String COMPLIANCEID_KEY = COMPLIANCE_PREFIX + ".purger.complianceId";
+  // Path to the compliance id in the dataset descriptor json object.
+  public static final String DATASET_DESCRIPTOR_FIELDPATH = COMPLIANCE_PREFIX + ".dataset.descriptor.fieldPath";
+  public static final String DATASET_DESCRIPTOR_CLASS = COMPLIANCE_PREFIX + ".dataset.descriptor.class";
+  public static final String DEFAULT_DATASET_DESCRIPTOR_CLASS = "gobblin.compliance.DatasetDescriptorImpl";
+
+  // Name of the table containing the compliance ids based on which purging will take place.
+  public static final String COMPLIANCE_ID_TABLE_KEY = COMPLIANCE_PREFIX + ".purger.complianceIdTable";
+
+  /**
+   * Configuration keys for the dataset descriptor which will be present in tblproperties of a Hive table.
+   */
+  public static final String DATASET_DESCRIPTOR_KEY = "dataset.descriptor";
```

Comment:
This key is the property name which will be present in tblproperties of a hive table. It does't need to have a COMPLIANCE_PREFIX.